src/server/scripts/Outland/zone_shadowmoon_valley.cpp

```diff
@@ -1429,7 +1453,74 @@ class npc_enraged_spirit : public CreatureScript
 
         void Reset() override { }
 
-        void EnterCombat(Unit* /*who*/) override { }
+        void EnterCombat(Unit* /*who*/) override
+        {
+            switch (me->GetEntry())
+            {
+                case NPC_ENRAGED_WATER_SPIRIT:
+                    _events.ScheduleEvent(EVENT_ENRAGED_WATER_SPIRIT, Seconds(0), Seconds(1));
```

Comment:
You realy sure about it? 0~1 sec? sniffs show it?

Reply:
I am not sure i don't have accurate data only guessed from fotage

Reply:
If you dont have access to retail (to sniff it)
ask someone on IRC if can do it.
We dont accept guessed values.

Reply:
I closed it because you rather have this broken instead of using some temporary values. Looks like It is impossible to sniff things from retail right now because patch 7.3 has updated the exe to a powerful encryption or so i was told on #Trinity IRC.

Reply:
It's not a temp fix, it's hack.
Shauren should make a new sniffer soon, he always do. Or maybe Aokromes has some old sniff about this creatures. Is not only me... TC Team dont accept guessed values...

Reply:
> We dont accept guessed values.

One half or more of scripts(SAI, CPP) has wrong guessed timers and scripting methods(how NPC will use ability, for example), so it's strange to hear.
Since TC replaced EAI by SAI some progress was made by ACID(EAI), but it never been ported to TC. Sometimes [you can find there](https://raw.githubusercontent.com/ACID-Scripts/WOTLK/master/acid_wotlk.sql) good scripts with timers from retail for some TBC locations(currently ACID has perfectly scripted Tanaris, for example, and some TBC heroic dungeons. While TC uses old ACID version).
Or you can made script based on other scrips.

Reply:
So this thing is only one what holds for back merge?

Reply:
old times where different, things changed with time, from when we don't had snifs to now.

Reply:
PS: even you don't have sniffer now, you can look at combat logs.

Reply:
I suggest to merge other code and leave this script hanging.